modules/distribution-zones/src/main/java/org/apache/ignite/internal/distributionzones/DistributionZoneManager.java

```diff
@@ -631,7 +820,15 @@ public void stop() throws Exception {
 
         logicalTopologyService.removeEventListener(topologyEventListener);
 
-        metaStorageManager.unregisterWatch(watchListener);
+        metaStorageManager.unregisterWatch(topologyWatchListener);
+        metaStorageManager.unregisterWatch(dataNodesWatchListener);
+
+        topVerTracker.update(Long.MAX_VALUE);
```

Comment:
Let's move it a constant value and add a comment, why Long.MAX_VALUE)

Reply:
Discussed in private. I added comment but I didn't create the constant.